packages/core/docs/changelog/5442_1.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'Added `is-authenticated` middleware to protect user profile routes from guest access. By default it emits an error message asking to implement vendor-specific validation.',
+  link: 'https://github.com/vuestorefront/vue-storefront/pull/5442',
+  isBreaking: true,
+  breakingChanges: [],
```

Comment:
missing before/after. If you include BC's please always put them in this table. It's easier to parse for integrators